### PR TITLE
Adelle/feature/500popup

### PR DESCRIPTION
**Pull request by Adelle-Pitsas**
# Description
This feature adds the Error handing for anything other than a 400 error. We have left it intentionally vague to deal with a variety of different errors. We have styled it a bit so we know it should be a pop-up modal.

When the dismiss button on the modal is click it will automatically route back to the top100 view or `'/'`

# Fixes # (issue)
#25 

# Type of change

- [ ]  Bug fix (non-breaking change which fixes an issue)
 
- [x]  New feature (non-breaking change which adds functionality)

- [ ]  Breaking change (fix or feature that would cause existing functionality not to work as expected)

- [ ] This change requires a documentation update

-  [x] Styling

- [ ] Refactor

- [ ] Testing

# Where were the changes?
Created new `Error.tsx` and `Error.css`. Files `App.tsx`, `Details.tsx`, and `game.ts`(in apiCalls) were touched in the process. 

# Checklist:

- [x]  My code follows the style guidelines of this project
- [x]  I have performed a self-review of my code
- [ ]  I have commented my code, particularly in hard-to-understand areas (if applicable)
- [ ]  I have made corresponding changes to the documentation (if applicable)
- [x]  My changes generate no new warnings

# Contributors (paired programming):
Ryan: @Nagel29 
Dmitrii: @dgubko

### CODE REVIEW

File: src/components/Error/Error.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { Link } from 'react-router-dom'
+import errorIcon from '../../images/alert-icon.png'
```
Comment (Courtney-Lippman) [inline src/components/Error/Error.tsx:2]: The error icon looks really good!

File: src/components/Details/Details.tsx [hunk 2]
```diff
@@ -7,21 +7,28 @@ import { BsSuitHeartFill } from "react-icons/bs"
 import './Details.css'
 
 
-const Details = ({ details, top100 }: { details?: GameDetails, top100: CleanedGame[] }) => {
+
+const Details = ({top100, updateError}: { top100: CleanedGame[], updateError: () => void  }) => {
```
Comment (Courtney-Lippman) [inline src/components/Details/Details.tsx:11]: The types look good!

**Review by Courtney-Lippman — APPROVED** [review]
Everything looks good! The error Icon is a great touch. All the types look good too.

**Review by Nagel29 — COMMENTED** [review]
I'm glad we figured out a way to do this with only having the error state live in App!

**Review by Nagel29 — APPROVED** [review]

**Pull request merged**